src/gei/BlobServiceClientFactory.cs

```diff
@@ -0,0 +1,14 @@
+using Azure.Storage.Blobs;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter
+{
+    public class BlobServiceClientFactory : IBlobServiceClientFactory
+    {
+        public BlobServiceClientFactory() { }
```

Comment:
We can remove the default constructor I guess
```suggestion
```

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/266f0d07425b14207ad953a1fb8661746c973ed1